javascript/javascript1/week1/main.js

```diff
@@ -0,0 +1,50 @@
+// Age-ify (A future age calculator)
+let yearOfBirth = 1996;
+let yearFuture = 2027;
+let age = yearFuture - yearOfBirth;
+console.log("You will be " + age + " years old in " + yearFuture + ".");
+
```

Comment:
great start! I only have one comment here and throughout your code, you seem to use exclusively let. Would it make sense to use const here? where? :)

Reply:
@vulk29 maybe it make sense to use const for date of birth it will not change :)